src/main/java/seedu/duke/Parser.java

```diff
@@ -110,6 +111,11 @@ public static Command checkAction(String inputCommand) throws DukeExceptions {
         case "members":
             commandType = new ListTeamMembersCommand();
             break;
+
+        case "remove":
+            memberName = inputCommand.split(" ")[1];
```

Comment:
Should use the inputs array that Riaz implemented where he split the inputCommand into multiple words already.